mLRS/CommonTx/mlrs-tx.cpp

```diff
@@ -752,7 +752,7 @@ RESTARTCONTROLLER
     sx_serial.Init(&serial, &mbridge, &serial2); // ports selected by SerialDestination, ChannelsSource
     cli.Init(&comport);
 #ifdef USE_ESP_WIFI_BRIDGE
-  #ifdef DEVICE_HAS_JRPIN5_FULL_DUPLEX
+  #if defined DEVICE_HAS_JRPIN5_NO_TC && UART_USE_TX_IO != UART_USE_RX_IO
```

Comment:
aha ... here there is a difference
I don't like that at all, super cryptic and things like the UART should really be used only low level. Just think about what happens when it's not the UART but a different UARTX ... no.
either:
- option a: this part is done in the hal.h and a respective USE define is set and used everywhere higher up
- option b: the DEVICE_JRPIN5_FULL_DUPLEX is kept
I guess I would prefer b as it is just easy to comprehend